src/System.Net.Security/tests/UnitTests/System.Net.Security.Unit.Tests.csproj

```diff
@@ -10,7 +10,7 @@
     <ProjectGuid>{0D174EA9-9E61-4519-8D31-7BD2331A1982}</ProjectGuid>
     <OutputType>Library</OutputType>
     <StringResourcesPath>../../src/Resources/Strings.resx</StringResourcesPath>
-    <NugetTargetMoniker>.NETStandard,Version=v1.3</NugetTargetMoniker>
+    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
```

Comment:
Made changes to the configuration of these tests. However, since the UnitTests were configured to tests internal methods and are testing newly exposed APIs in NetStandard1.7, it didn't make sense to me to test them against netstandard1.3 or net46 anymore. @stephentoub any thoughts on this?